Documentation/Documentation/Rendering/Index.rst

```diff
@@ -0,0 +1,27 @@
+.. include:: ../../Includes.txt
+
+.. _documentation:
+
+==============
+Documentation rendering
+==============
+
+.. _how-to-render-documentation-locally:
+
+How to render documentation locally?
+================
+
+In order to render documentation, first of all, clone repository:
+
+.. code-block:: none
+
+   git clone https://github.com/TYPO3-Documentation/tea.git
+
+then go to extension root:
+
+.. code-block:: none
+
+   cd tea
+
+
+and follow `this guide <https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/RenderingDocs/Quickstart.html>`_
```

Comment:
Maybe we can add a more descriptive title, e.g., "the TYPO3 documentation quickstart guide".